tasks.py

```diff
@@ -783,6 +792,7 @@ def assets(dev=False, watch=False):
     # on prod
     webpack(clean=False, watch=watch, dev=dev)
 
+
```

Comment:
This is fine, but FWIW, we explicitly ignore the 2-line rule: https://github.com/CenterForOpenScience/osf.io/blob/9ca27b0c5264a97a299fa4fd1c69f3a7914d0365/setup.cfg#L6. It adds unnecessary whitespace without affecting readability at all.